### ISSUE TITLE
Is it possible to forcefully enable Cinema Screen on LG C1?

### ISSUE THREAD

**Issue by durrburger93** [Maassoft/ColorControl#148]
Love this app but just wanted to know whether it's possible to enable the Cinema Screen mode while using PC, as this setting perfectly removes any judder from videos but is unavailable when PC is connected. It's one of the rare options that's not available in Color Control from what I'm seeing, though I could be wrong and it could be named as something else. Thanks!

**Comment by Maassoft** [comment]
Yes, it is possible, but the option has not been added to Color Control yet.
But I don't know if it's doing anything while in PC mode. If you play a movie on your PC, the TV wouldn't know at what framerate the movie plays, unless you switch to 24Hz for example. I think it only works as it should when playing movies/series via internal apps on the TV itself.

**Comment by chros73** [comment]
TruMotion setting doesn't work on 2022 models, although the values are valid hut doesn't change the option, so it's a bug.
